### PR TITLE
Secret generator: Add validate-only flag to allow running in presubmit

### PR DESCRIPTION
**Pull request by alvaroaleman**
/cc @openshift/openshift-team-developer-productivity-test-platform

### CODE REVIEW

File: cmd/ci-secret-generator/main.go [hunk 1]
```diff
@@ -57,6 +58,7 @@ func parseOptions() options {
 	flag.CommandLine.StringVar(&o.configPath, "config", "", "Path to the config file to use for this tool.")
 	flag.CommandLine.StringVar(&o.bootstrapConfigPath, "bootstrap-config", "", "Path to the config file used for bootstrapping cluster secrets after using this tool.")
 	flag.CommandLine.BoolVar(&o.validate, "validate", true, "Validate that the items created from this tool are used in bootstrapping")
+	flag.CommandLine.BoolVar(&o.validateOnly, "validate-only", false, "If the tool should exist after the validation")
```
Comment (stevekuznetsov) [inline cmd/ci-secret-generator/main.go:61]: `s/exist/exit/`

Comment (alvaroaleman) [reply cmd/ci-secret-generator/main.go:61]: But it will stop existing after the validation!

Comment (alvaroaleman) [reply cmd/ci-secret-generator/main.go]: updated

**Review by petr-muller — APPROVED** [review]

**Pull request merged**